File: C++11/109.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int n;
    cin >> n;

    int g = n % 10;
    int s = n / 10 % 10;
    int b = n / 100 % 10;
    int q = n / 1000 % 10;

    if (!(g % 2) ||!(s % 2) ||!(b % 2) ||!(q % 2)) cout << "YES" << endl;
    else cout << "NO" << endl;

    return 0;   
}

File: Python3/109.py
n = int(input())
g = n % 10
s = n // 10 % 10
b = n // 100 % 10
q = n // 1000
if g % 2 == 0 or s % 2 == 0 or b % 2 == 0 or q % 2 == 0:
    print("YES")
else:
    print("NO")